build.gradle

```diff
@@ -82,7 +82,8 @@ ext {
     junitPlatformConsole: "org.junit.platform:junit-platform-console",
     log4j               : "log4j:log4j:1.2.17",
     objenesis           : "org.objenesis:objenesis:3.3",
-    jacocoAgent         : "org.jacoco:org.jacoco.agent:$jacocoVersion:runtime"
+    jacocoAgent         : "org.jacoco:org.jacoco.agent:$jacocoVersion:runtime",
+    discordWebhooks     : "club.minnced:discord-webhooks:0.8.2"
```

Comment:
Not sure if its worth to pull in this and its dependencies for this.
We already have `testCompileOnly libs.jetbrainsAnnotations` so maybe add a class to the spock-specs `main` source set that uses them.

Reply:
Ok, after much fiddling around I was able to reproduce the error with just using JB annotations.